app/src/main/java/org/kiwix/kiwixmobile/utils/ImageUtils.kt

```diff
@@ -0,0 +1,78 @@
+package org.kiwix.kiwixmobile.utils
+
+import android.graphics.Bitmap
+import android.graphics.Bitmap.createBitmap
+import android.media.ThumbnailUtils
+import android.view.View
+import android.view.View.MeasureSpec
+
+object ImageUtils {
+  @JvmStatic
+  fun getBitmapFromView(
+    viewToDrawFrom: View,
+    width: Int,
+    height: Int
+  ): Bitmap? {
+    val previousDrawingCacheState = viewToDrawFrom.isDrawingCacheEnabled
+    viewToDrawFrom.isDrawingCacheEnabled = true
```

Comment:
Some of the functions like `isDrawingCacheEnabled` are deprecated. I don't what this to be reverted but I want to discuss some alternative approaches to this. Should we render the WebViews as bitmaps or is there any other way to render the tabs in the tabs overview screen? I tried to find some libraries which can give us Google Chrome-like tabs. I found this library https://github.com/michael-rapp/ChromeLikeTabSwitcher. Another approach is to display tabs in titles like in Firefox. Currently, we only show a single tab in the overview screen but using tiles we can show multiple tabs. What could be the best approach?

Reply:
I don't see a reason why we can't just have the webviews in the TabsAdapter and have then just load their content when they are paged in. Bitmaps seem like a bit of overkill

Reply:
@macgills We can have WebViews, then we have to capture touch events when the tabs overview screen is displayed so that the touch for selecting the WebView is not propagated to the WebView which may have a href and the user is navigated to another page when the tab overview screen is shown.

We also have to handle the resizing of the WebViews. IMO we should try this approach.

Reply:
It has my vote. Want to create a ticket?